patches/chrome-installer-util-util_constants.cc.patch

```diff
@@ -1,17 +1,24 @@
 diff --git a/chrome/installer/util/util_constants.cc b/chrome/installer/util/util_constants.cc
-index e8913e34bd4731886c2709db051ed9a0bf3115ac..aa7779e71a9772cd628ed20835a2d7ca0c56c7f7 100644
+index e8913e34bd4731886c2709db051ed9a0bf3115ac..2b2456a0f634bfed591bb88c0a541ed56f66e257 100644
 --- a/chrome/installer/util/util_constants.cc
 +++ b/chrome/installer/util/util_constants.cc
-@@ -184,9 +184,9 @@ const char kGoogleUpdateIsMachineEnvVar[] = "GoogleUpdateIsMachine";
+@@ -182,11 +182,19 @@ const char kGoogleUpdateIsMachineEnvVar[] = "GoogleUpdateIsMachine";
+ // TODO(gab): Rename setup.exe itself altogether and use the same binary for
+ // Active Setup.
  const wchar_t kActiveSetupExe[] = L"chrmstp.exe";
- const wchar_t kChromeDll[] = L"chrome.dll";
- const wchar_t kChromeChildDll[] = L"chrome_child.dll";
--const wchar_t kChromeExe[] = L"chrome.exe";
--const wchar_t kChromeNewExe[] = L"new_chrome.exe";
--const wchar_t kChromeOldExe[] = L"old_chrome.exe";
++#if defined(BRAVE_CHROMIUM_BUILD)
++const wchar_t kChromeDll[] = L"brave.dll";
++const wchar_t kChromeChildDll[] = L"brave_child.dll";
```

Comment:
Same thing here with chrome/brave. Unless there is a compelling reason to rename these dlls I don't think we should